README.md

```diff
@@ -1 +1,171 @@
-# math-magician
\ No newline at end of file
+<a name="readme-top"></a>
+
+<div align="center">
+  <br/>
+
+  <h3><b></b></h3>
+
+</div>
+
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+  - [Usage](#usage)
+  - [Run tests](#run-tests)
+  - [Deployment](#deployment)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [📝 License](#license)
+
+
+# 📖 Math Magician <a name="about-project"></a>
+
+> In this project I will develop a user-friendly single-page application for performing various math calculations, but for this task I setup all the tool needed to start building my project
+
+## Built With
+
+- HTML
+- CSS
+- JAVASCRIPT
+- REACT
+
+### Tech Stack <a name="tech-stack"></a>
+
+> 
+
+<details>
+  <summary>Cient</summary>
+  <ul>
+    <li><a href="https://developer.mozilla.org/en-US/docs/Web/HTML">HTML</a></li>
+    <li><a href="https://developer.mozilla.org/en-US/docs/Web/CSS">CSS</a></li>
+    <li><a href="https://developer.mozilla.org/en-US/docs/Learn/JavaScript">Javascript</a></li>
+    <li><a href="https://react.dev/">React</a></li>
+  </ul>
+</details>
+
+
+### Key Features <a name="key-features"></a>
+
+- **Sngle Page Website**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+To get a local copy up and running, follow these steps.
+
+### Prerequisites
+
+In order to run this project you need the following tools:
+- Node.js on your computer
+- Code editor e.g.: VScode
+
+### Setup
+
+Clone this repository to your desired folder by running the following commands in your terminal:
+
+```sh
+  cd your-prefered-folder-name
+  
+  git clone https://github.com/mershark/math-magician.git
+```
+
+### Install
+
+- Install this project with:
+
+```sh
+  cd math-magician
+
+  npm install or npm i
+```
+
+### Usage
+
+To run the project, execute the following command:
+
+```sh
+  npm start
+
+```
+
+### Run tests
+
+To run tests, run the following command:
+
+```sh
+  npm run test
+```
+
+### Deployment
+
+You can deploy this project by transferring the files in your local repository onto your server
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- VIDEO WALKTHROUGH -->
+
+## 🚀 Video Walkthrough <a name="live-demo"></a>
+
+([coming soon]())
+
+
+<!-- LIVE DEMO -->
+
+## 🚀 Live Demo Website <a name="live-demo"></a>
+
+([coming soon]())
+
```

Comment:
- [ ] As this makes your README file not more professional So I would suggest please remove this `video and live Demo `link as you did not have a link right now. In the future, if you decide to put the link you can put it with a video and live demo link. So for now please remove it. It make more descriptive README file.